external/common/include/fast_cpp_csv_parser/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2015, ben-strasser
```

Comment:
Just call it `README.md` and we can put details for any of the libraries in the same file. The nice thing about that is that it will show the README.md file when you are in that directory in github.